dbms/src/Storages/MergeTree/MergeTreeIndices.h

```diff
@@ -143,4 +143,24 @@ class MergeTreeIndexFactory : public ext::singleton<MergeTreeIndexFactory>
     Indexes indexes;
 };
 
+std::unique_ptr<IMergeTreeIndex> minmaxIndexCreator(
```

Comment:
Maybe better to make functions static / in anonymous namespace (internal linkage) instead of exposing in header?